src/lib/cli.js

```diff
@@ -18,16 +19,19 @@ program
   .option('-r, --require <path>', 'Require location for support files (repeatable)', collect, [])
   .option('-c, --cucumber <path>', 'Path to specific cucumber installation')
   .option('-w, --workers <int>', 'Number of workers (Defaults to # Processors)', parseInt)
-  .option('-l, --logdir <path>', 'Output dir for test logs')
-  .option('-s, --silentsummary', 'Silences summary output so it can be handled via the returned promise')
+  .option('-l, --log-dir <path>', 'Output dir for test logs. Default: ' + defaultOptions.logDir)
+  .option('-s, --silent-summary', 'Silences summary output so it can be handled via the returned promise')
   .option('-v, --verbose', 'Adds verbose output to console')
-  .option('-i, --inlinestream',
+  .option('-i, --inline-stream',
     'Inlines stream in real time in addition to multi-cuke output. ' +
     '*Note* This adds complexity to the logs that are hard to decipher, but included if needed for debugging'
   )
-  .option('-d, --devMode', 'Shortcut for running cucumber-js directly')
+  .option('-d, --dev-mode', 'Shortcut for running cucumber-js directly')
   .option('--fail-fast', 'abort the run on first failure')
   .option('--strict', 'Fail fast if a step is undefined')
+  .option('--merged-log <path>', 'Path for merged log. Specify "" to not generate one. ' +
```

Comment:
It feels like we're trying to overload the path with also handling a 'should-generate-logs' boolean sort of flag by the use of omiting a path here. I think it'd probably be cleaner to provide both the `--merged-log` flag along with something like `--disable-merged-log` or `--no-merged-log`